File: Cmr-Tc/04-08-23/Frequency.java
import java.util.HashMap;

class Frequency{
    public static void main(String[] args) {
        HashMap<Integer,Integer> map = new HashMap<>();
        int[] arr = {-2,4,6,4,87,56,45,87,45,35};  
        for(int e: arr){
            if(map.containsKey(e)){
                int value = map.get(e);
                map.put(e,value+1);
            }else{
                map.put(e,1);
            }
        }
        System.out.println(map);
    }
}

File: Cmr-Tc/04-08-23/frequency.py
def main():
    arr = [1,1,1,2,3,32,2,3,4,3,34,2]
    ans = [0 for i in range(101)]
    for e in arr:
        ans[e]+=1
    for i in range(len(ans)):
        if ans[i]>0:
            print(i,"->",ans[i])

main()
